bin/release

```diff
@@ -16,7 +16,7 @@ EOF
 if [[ $MANAGE_FILE ]]; then
 cat <<EOF
 
-addons:
-  - heroku-postgresql
+#addons:
+#  - heroku-postgresql
```

Comment:
adding heroku-postgresql adddon has been commented out, but looks like we were using it earlier.

Reply:
No we weren't :) 
Exhibit A: `bin/release` on the [old master branch](https://github.com/atzannes/heroku-buildpack-python-old/blob/master/bin/release) and on the current [legacy branch](https://github.com/AltSchool/heroku-buildpack-python/blob/legacy/bin/release)

Exhibit B: the [commit](https://github.com/atzannes/heroku-buildpack-python-old/commit/7493d1b06af1703d49a46d29097493ad0398f917) that made this change originally

Reply:
👍